src/Microsoft.DotNet.CodeAnalysis/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "dependencies": {
+    "Microsoft.Net.Compilers.NetCore": "1.2.0-beta1-20160202-02",
```

Comment:
Does this need to be this NetCore compiler package? Why not the Microsoft.CodeAnalysis.CSharp package?